src/test/java/org/openmrs/contrib/qaframework/helper/RestClient.java

```diff
@@ -0,0 +1,142 @@
+package org.openmrs.contrib.qaframework.helper;
+
+import java.io.IOException;
+
+import org.glassfish.jersey.client.authentication.HttpAuthenticationFeature;
+import org.openmrs.contrib.qaframework.helper.TestData.JsonTestClass;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.client.ClientBuilder;
+import jakarta.ws.rs.client.Entity;
+import jakarta.ws.rs.client.WebTarget;
+import jakarta.ws.rs.core.MediaType;
+
+public class RestClient {
```

Comment:
Hey! This is exactly the kind of thing we can leverage for REST API tests...

Reply:
Are you saying we mantain this in distro ?